experimental/logging/log_listener_example_test.go

```diff
@@ -19,10 +19,11 @@ import (
 //go:embed testdata/listener.wasm
 var listenerWasm []byte
 
-// This is a very basic integration of listener. The main goal is to show how it is configured.
-func Example_newLoggingListenerFactory() {
+// This is a very basic integration of listener. The main goal is to show how
+// it is configured.
+func Example_newHostLoggingListenerFactory() {
 	// Set context to one that has an experimental listener
-	ctx := context.WithValue(context.Background(), experimental.FunctionListenerFactoryKey{}, logging.NewLoggingListenerFactory(os.Stdout))
+	ctx := context.WithValue(context.Background(), experimental.FunctionListenerFactoryKey{}, logging.NewHostLoggingListenerFactory(os.Stdout))
```

Comment:
so this is using `logging.NewHostLoggingListenerFactory`, and it doesn't show the intermediate guest function.